### ISSUE TITLE
Making a new release

### ISSUE THREAD

**Issue by pkgw** [guillochon/MOSFiT#228]
Hi all —

Over at conda-forge, we're running a migration to build a new package of mosfit for Python 3.12. [The build isn't working][1] because of the `np.float` deprecation, which is fixed in Git but hasn't yet made it into a new release. Would you be able to put one out? There are workarounds that we can do, but for the packaging it makes a big difference to have an official release as a baseline.

[1]: https://github.com/conda-forge/mosfit-feedstock/pull/53

**Comment by mnicholl** [comment]
Thanks Peter. This has prompted me to check in on a couple of bug fixes that were in the works -- once those are done I'm happy to do a release. Should I also bump the version number to differentiate from what's currently on pip/conda?

**Comment by pkgw** [comment]
Yes, absolutely. Once you put out a release at a specific version number, the associated artifacts (source code files, compiled binaries, etc.) should never change. Ideally the version numbering would follow [semver](https://semver.org/) which would probably mean that the next version would be 1.1.10.
